exporter/trace/stackdriver/stackdriver.go

```diff
@@ -31,6 +31,7 @@ package stackdriver
 
 import (
 	"context"
+	"errors"
```

Comment:
Trace is not affected by this problem. It is only the stats exporter.

Reply:
How come trace isn't affected?

Reply:
Stackdriver Monitoring has a requirement that the timelines need to be uploaded in a particular order. That makes us always upload data in chronological order. If one exporter uploads data earlier than the other one, it will be rejected by the API endpoint.

Stackdriver Trace doesn't have the same restriction.